gnucash/gnome/window-reconcile.c

```diff
@@ -250,6 +250,16 @@ recn_get_account (RecnWindow *recnData)
 }
 
 
+static void
+WidgetSetAmount (GtkWidget *widget, gnc_numeric amt, GNCPrintAmountInfo print_info,
+                 gboolean reverse)
```

Comment:
Except for the old xaccFooDoBar api we should use lower-case-with-underscores for function names, reserving camel case for types. Also, better to be more specific than `widget`. How about `label_set_amount` or `label_add_amount_string_and_color`?

Reply:
`gnc_add_colorized_amount`, and can take any gobject.